src/storage.ts

```diff
@@ -269,11 +269,15 @@ class AnnotationServerStorage {
      * @param {string} targetUri URI of the target to search for
      */
     async search(targetUri: string): Promise<void | SavedAnnotation[]> {
-        const { annotationEndpoint, sourceUri, manifest } = this.settings;
-        const sourceQuery = sourceUri ? `&source=${sourceUri}` : "";
-        const manifestQuery = manifest ? `&manifest=${manifest}` : "";
+        const { annotationEndpoint, sourceUri, manifest, secondaryMotivation } =
+            this.settings;
+        const sourceQ = sourceUri ? `&source=${sourceUri}` : "";
+        const manifestQ = manifest ? `&manifest=${manifest}` : "";
+        const motivationQ = secondaryMotivation
```

Comment:
will secondary motivation always be set? is it strictly required?

Reply:
Tahqiq will just ignore it, and not pass it on to this search query, if it's not provided—though it is required for Geniza to properly differentiate between transcription and translation. Should it be required here too?

Reply:
I think it's better if it's not required here. I see now that's exactly what it's doing, so this looks good to me.